tailwind.config.js

```diff
@@ -19,6 +19,7 @@ try {
 let themeColors = {
   primary: uiColors.indigo,
   secondary: uiColors.purple,
+  'cat-foobar': '#ffffff'
```

Comment:
I have to declare `cat-foobar` here in order for it to be picked up in `theme/tailwind.config.yml`. The value of `cat-foobar` in `theme/tailwind.config.yml` will override whatever is here

Reply:
This shouldn't be the case. You should be able to declare new colors only in the theme/ tailwind file. You should look over @adrianthedev's code that loads this to investigate why that's happening.